examples/evm-to-evm-generic-message-transfer/src/transfer.ts

```diff
@@ -0,0 +1,52 @@
+import dotenv from "dotenv";
+import { createCrossChainContractCall } from '@buildwithsygma/evm';
+import { Wallet, ethers, providers } from "ethers";
+import { storageAbi } from './contracts';
+import { Eip1193Provider } from "@buildwithsygma/core";
+import Web3HttpProvider from "web3-providers-http";
+
+dotenv.config();
+
+const privateKey = process.env.PRIVATE_KEY;
+
+if (!privateKey) {
+  throw new Error("Missing environment variable: PRIVATE_KEY");
+}
+
+const DESTINATION_CHAIN_ID = 17000;
```

Comment:
it is hardcoded in other examples, what I can do is move all of them to env variables, however, its more descriptive here as we are guiding someone about the SDK. Wdyt?

Reply:
no, you are right, let's follow the same pattern

Reply:
As we use a constant it would be nice to mention to which network the chain ID belongs to, f.e.
```
const  Holesky_DESTINATION_CHAIN_ID = 17000
```
or as a comment
```
const  DESTINATION_CHAIN_ID = 17000 // Holesky
```

Reply:
Btw in the docs, the destination network is Base-Sepolia https://docs.buildwithsygma.com/sdk/examples/gmp/evm-example-gmp/#gmp-example-with-a-simple-storage-contract

Reply:
@Lykhoyda I will update this, thanks :)